### DISCUSSION TITLE
1pux import error

### DISCUSSION THREAD

**Discussion by mik0s** [dani-garcia/vaultwarden#3129] [General]
I have 1pux (1password version 8.9.11) with about 2000 entries (passwords, accounts, notes, credit cards). I'm trying to import it into vaultvarden (version 2022.12.0).

After clicking the "Import" button, I do not see any messages about the end of the import or errors. I see that some data has appeared in Vaultwarden, but not all.

I went into the logs of the docker container and at the time of import I see the following entries:
```
[2023-01-09 17:05:07.930][request][INFO] POST /api/ciphers/import
[2023-01-09 17:05:08.418][vaultwarden::api::core::ciphers][ERROR] The field Notes exceeds the maximum encrypted value length of 10000 characters.
[2023-01-09 17:05:08.433][response][INFO] (post_ciphers_import) POST /api/ciphers/import => 400 Bad Request
```

How do I import my 1pux?

**Comment by BlackDex** [comment]
There already is a patch for that, but that is not yet merged into a newer version of Vaultwarden.
You need to figure out which ciphers have notes which exceed the limit support by Vaultwarden/Bitwarden.

If the patch is merged it would show you which specific items.
For now, the only way to see that is create a free account on vault.bitwarden.com and try an import there. It will notify you of which ciphers have issues.

**Reply by mik0s** [reply]
Thank you. How long does it take for this patch to become available do you think?

**Reply by BlackDex** [reply]
With a bit of luck, very soon :)

**Reply by BlackDex** [reply]
And with that said.
The build is currently in progress: https://github.com/dani-garcia/vaultwarden/actions/runs/3876986155

After that, the `testing` tagged release will be available with the needed fixes.

**Reply by mik0s** [reply]
Thank you. It helped me.
